docs/sources/clients/promtail/stages/match.md

```diff
@@ -5,14 +5,18 @@ description: match stage
 # match
 
 The match stage is a filtering stage that conditionally applies a set of stages
-or drop entries when a log entry matches a configurable [LogQL]({{< relref "../../../query" >}})
-stream selector and filter expressions.
+or drop entries when a log entry matches a configurable LogQL
+[stream selector]({{< relref "../../../query/log_queries#log-stream-selector" >}}) and
+[filter expressions]({{< relref "../../../query/log_queries#line-filter-expression" >}}).
+
+Please note that the filters do not include label filter expressions such as `|
+label == "foobar"`.
```

Comment:
```suggestion
```suggestion
{{% admonition type="note" %}}
The filters do not include label filter expressions such as `| label == "foobar"`.
{{% /admonition %}}



```